frontend/task/libs/smart_contract/smart_contract_lib.ts

```diff
@@ -141,7 +151,12 @@ export class SmartContractLib extends QuickAlgoLibrary {
             }
 
             return new LibraryTestResult(
-                output.error?.message
+                output.error?.message,
+                'task-submission-test-result-smart-contract',
```

Comment:
What is this part for? I don't see where it's used

Reply:
:+1: